Source/ThirdParty/ANGLE/Configurations/ANGLE-dynamic.xcconfig

```diff
@@ -33,8 +33,7 @@ GCC_PREPROCESSOR_DEFINITIONS = $(DEBUG_DEFINES) LIBGLESV2_IMPLEMENTATION LIBANGL
 GCC_PREPROCESSOR_DEFINITIONS_macosx = ANGLE_ENABLE_GLSL ANGLE_ENABLE_ESSL ANGLE_ENABLE_OPENGL ANGLE_ENABLE_GL_DESKTOP_BACKEND ANGLE_ENABLE_APPLE_WORKAROUNDS ANGLE_ENABLE_CGL GL_SILENCE_DEPRECATION
 GCC_PREPROCESSOR_DEFINITIONS_maccatalyst = $(GCC_PREPROCESSOR_DEFINITIONS_macosx)
 GCC_PREPROCESSOR_DEFINITIONS_maccatalyst[arch=arm*] = $(GCC_PREPROCESSOR_DEFINITIONS_macosx) ANGLE_ENABLE_EAGL GLES_SILENCE_DEPRECATION ANGLE_CPU_ARM64
-// FIXME: Rest of the iOS family does not need EAGL, these will be removed when upstream is able to compile without it.
-GCC_PREPROCESSOR_DEFINITIONS_iphoneos = ANGLE_ENABLE_GLSL ANGLE_ENABLE_ESSL ANGLE_ENABLE_OPENGL ANGLE_ENABLE_APPLE_WORKAROUNDS ANGLE_ENABLE_EAGL GLES_SILENCE_DEPRECATION
+GCC_PREPROCESSOR_DEFINITIONS_iphoneos =
```

Comment:
We don't need an empty definition (nor the lines below, these should all be removed now). They do nothing other than inviting questions like "where's visionOS?"

Secondly, the name is not great - `GCC_PREPROCESSOR_DEFINITIONS_maccatalyst` implies that this is the whole value of `GCC_PREPROCESSOR_DEFINITIONS` for the platform, not just platform-specific ANGLE additions.